### PR TITLE
Add documentation sexism to life/gender

### PR DESCRIPTION
**Pull request by laurpaik-zz**
re: marliave talk the other day

### CODE REVIEW

File: life/gender/documentation-sexism.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Documentation can be sexist
+
+This is one of those things that never occurred to me, but once someone pointed
+it out I realized I'm not that surprised. (Thanks @raq929)
+
+Part of it's that I'm new to tech, so a lot of the examples I've seen were
```
Comment (BenGitsCode) [inline life/gender/documentation-sexism.md]:
```diff
- Part of it's that I'm new to tech, so a lot of the examples I've seen were
+ Part of it is that I'm new to tech, so a lot of the examples I've seen were
```

**Review by BenGitsCode — COMMENTED** [review]
I just saw a quick grammar fix. Not a big deal. (not even positive that's incorrect but the contraction feels weird)

Maybe consider adding this though https://guides.emberjs.com/v2.11.0/object-model/bindings/

This is the first documentation I saw that was like woah, not cool. If you don't read into it maybe it's not that bad, but if you think about the example it's really gross and there are much better ways to do demonstrate that example.

File: life/gender/documentation-sexism.md (hunk 1 above)
Comment (BenGitsCode) [reply life/gender/documentation-sexism.md]: Actually, that's all found so I'm just going to edit after merging. I don't think it's worth adding the ember docs link unless you want to, I mostly wanted you to just see it.

**Review by BenGitsCode — APPROVED** [review]

File: life/gender/documentation-sexism.md (hunk 1 above)
Comment (laurpaik-zz) [reply life/gender/documentation-sexism.md]: lmao that's a pretty good bad example. I think it's worth adding lol or even just screenshotting it for kicks

Comment (BenGitsCode) [reply life/gender/documentation-sexism.md]: Screenshotting for kicks. I like that.

**Review by BenGitsCode — APPROVED** [review]
+1 for commit name and another +1 for uploading it to aws.

**Pull request merged**